extensions-core/google-extensions/src/main/java/org/apache/druid/storage/google/GoogleStorageDruidModule.java

```diff
@@ -43,6 +45,7 @@
 public class GoogleStorageDruidModule implements DruidModule
 {
   static final String SCHEME = "google";
+  static final String SCHEME_GS = "gs";
```

Comment:
there is also `GoogleCloudStorageInputSource.SCHEME`, and `GoogleUtils.deleteObjectsInPath` which is just using `"gs"` instead of a static variable, could those be consolidated to use this one?

Reply:
ok missed that, changed everything to use `GoogleStorageDruidModule.SCHEME_GS`